test/cli.test.js

```diff
@@ -102,6 +93,27 @@ it('shows size-limit content error', () => {
   })
 })
 
+it('uses ".size-limit" file config', () => {
+  return run([], { cwd: fixture('with-file-config') }).then(result => {
+    expect(result.out).toEqual('\n' +
+    '  Package size: 20 B\n' +
+    '  Size limit:   1 KB\n' +
+    '  With all dependencies, minified and gzipped\n' +
+    '\n')
+    expect(result.code).toEqual(0)
+  })
+})
+
+it('Wrong ".size-limit" file config', () => {
+  return run([], { cwd: fixture('wrong-file-config') }).then(result => {
+    expect(result.out).toContain(
+      ' ERROR  Can not parse Size Limit config: \n' +
+      'missed comma between flow collection entries'
```

Comment:
All other error messages have indent on next lines.

```
 ERROR  Can not parse Size Limit config:
        missed comma between flow collection entries
```

Reply:
Will fix it